website/docs/configuration/variables.html.md

```diff
@@ -134,35 +148,42 @@ commentary for module maintainers, use comments.
 When variables are declared in the root module of your configuration, they
 can be set in a number of ways:
 
-* Individual assignments made on the command line.
-* Variable definitions files, either specified on the command line or
-  automatically loaded.
-* Environment variables.
+* [In a Terraform Enterprise workspace](/docs/enterprise/workspaces/variables.html).
+* Individually, with the `-var` command line option.
+* In variable definitions (`.tfvars`) files, either specified on the command line
+  or automatically loaded.
+* As environment variables.
 
 The following sections describe these options in more detail. This section does
 not apply to _child_ modules, where values for input variables are instead
 assigned in the configuration of their parent module, as described in
-[_Modules_](/docs/configuration/modules.html).
+[_Modules_](./modules.html).
 
 ### Variables on the Command Line
 
-To specify individual modules on the command line, use the `-var` argument
-that is accepted by the `terraform plan` and `terraform apply` commands:
+To specify individual modules on the command line, use the `-var` option
+when running the `terraform plan` and `terraform apply` commands:
 
 ```
 terraform apply -var="image_id=ami-abc123"
 ```
 
-### Variable Definitions Files
+The `-var` argument can be used any number of times in a single command.
```

Comment:
Since you changed "argument" to "option" above (which makes sense to me!) we should probably change this one too.